ipa-core/src/protocol/ipa_prf/malicious_security/verifier.rs

```diff
@@ -8,38 +8,35 @@ use crate::{
 };
 
 #[allow(non_upper_case_globals)]
-pub struct ProofVerifier<F: PrimeField> {
-    zero_share: F,
-}
+pub struct ProofVerifier {}
```

Comment:
there may be some further work on this abstraction - I can't say exactly what is going on here, but having a lot of methods that don't take `self` as an argument is suspicious

Reply:
It's just a collection of static methods. There is no need for any struct, just some place to put them.

Reply:
I would say it is not idiomatic in Rust to keep stateless structs - OOP went with it because everything has to be a class, but in Rust we can just use standalone functions. structs usually represent some sort of a state, either at compile time or at runtime. 

I was thinking that we are probably missing some abstractions here, but I can't recommend anything at this point.